tests/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/onflow/flow-evm-gateway/integration
 
-go 1.20
+go 1.22
```

Comment:
We probably don't want these changes to go in. Including the line:
```go
toolchain go1.22.3
```

Reply:
my goland keeps updating this in a loop I can't seem to get it back, honestly I don't have the time now to deal with this, if you want you can downgrade and let me know how to fix it. sorry

Reply:
I've tried many things. I don't see harm in using go 1.22

Reply:
I see now the reason the rate limiter library I use requires v1.22.